docs/python/training/content.rst

```diff
@@ -0,0 +1,54 @@
+This document describes ORTModule PyTorch frontend API for the ONNX Runtime (aka ORT) training acelerator.
```

Comment:
Will this show up in any of our websites?

Should we add a note to this file to inform readers that the ORTModule is currently under development, and the API may change?

Reply:
Not yet. this first PR only introduces all the tooling required to produce python API doc from a build. This content is a mere placeholder for content to come

Next PRs will actually include all relevant content